src/routes/bitbucket/proxy/index.ts

```diff
@@ -180,6 +181,17 @@ export function proxyRoutes(runner: Runner, client: BitbucketClient) {
         targetBranch: prInfo.targetBranch,
       });
 
+      const priority = await client.bitbucket.getPullRequestPriority(commit);
+
+      if (priority === 'HIGH' && request) {
+        await request.incrementPriority();
+        Logger.info('Pull request priority increased', {
+          namespace: 'routes:bitbucket:proxy:land',
+          pullRequestId: prId,
+          landRequest,
+        });
+      }
```

Comment:
An alternative to incrementing priority after queueing would be to set the priority on land request creation. This works though.